spec/moonshot/dynamic_template_spec.rb

```diff
@@ -4,7 +4,7 @@
   include FakeFS::SpecHelpers
 
   let(:source_path) { 'spec_template.erb' }
-  let(:parameters) { { test_param: 'test-value' } }
+  let(:parameters) { -> environment_name { parameters = { test_param: 'test-value' } } }
```

Comment:
please add specs for the case when parameters is a Hash.

Reply:
Done. Thanks.